<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputPDFClass</class>
 <widget class="QMainWindow" name="OutputPDFClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OutputPDF</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">font-family:'Consolas';
font-size:14px;
background:#272822;
color:white;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
border:1px solid #409EFF;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_filename">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;
border-radius:2px;</string>
         </property>
         <property name="text">
          <string>NewPDF</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QToolButton" name="toolButton_openfolder">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:white;
color:black;
border:none;
border-top-right-radius:2px;
border-bottom-right-radius:2px;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_pageorientation">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:white;
color:black;</string>
         </property>
         <item>
          <property name="text">
           <string>纵向</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>横向</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>.pdf</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_pagesize">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:white;
color:black;</string>
         </property>
         <item>
          <property name="text">
           <string>A4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3Extra</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A4Extra</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A4Plus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A4Small</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A5Extra</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B5Extra</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JisB10</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_savefolder">
         <property name="text">
          <string>保存位置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_savefolder">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;
border-top-left-radius:2px;
border-bottom-left-radius:2px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_mode">
         <property name="text">
          <string>页面模式</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_pagesize">
         <property name="text">
          <string>页面大小</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_mode">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:white;
color:black;</string>
         </property>
         <item>
          <property name="text">
           <string>FullPageMode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>StandardMode</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pageorientation">
         <property name="text">
          <string>页面方向</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_filename">
         <property name="text">
          <string>文件名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_2{
border:1px solid #409EFF;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_output">
         <property name="styleSheet">
          <string notr="true">#pushButton_output{
border-radius:2px;
border:1px solid;
background:rgb(28.3, 37.4, 23.8);
border-color:rgb(61.5, 107, 39);
color:#67C23A;
}

#pushButton_output::hover{
background:#67C23A;
border-color:#67C23A;
color:white;
}

#pushButton_output::pressed{
background:rgb(133.4, 206.2, 97.4);
border-color:rgb(133.4, 206.2, 97.4);
color:white;
}</string>
         </property>
         <property name="text">
          <string>确定(Enter)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="styleSheet">
          <string notr="true">#pushButton_cancel{
border-radius:2px;
border:1px solid;
background:rgb(32.4, 32.7, 33.3);
border-color:rgb(82, 83.5, 86.5);
color:#909399;
}

#pushButton_cancel::hover{
background:#909399;
border-color:#909399;
color:white;
}

#pushButton_cancel::pressed{
background:rgb(166.2, 168.6, 173.4);
border-color:rgb(166.2, 168.6, 173.4);
color:white;
}</string>
         </property>
         <property name="text">
          <string>取消(Esc)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
